website/docs/r/servicebus_namespace_authorization_rule.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_servicebus_namespace_authorization_rule"
+sidebar_current: "docs-azurerm-resource-servicebus-namespace-authorization-rule"
+description: |-
+  Manages a ServiceBus Namespace authorization Rule within a ServiceBus Topic.
+---
+
+# azurerm_servicebus_namespace_authorization_rule
+
+Manages a ServiceBus Namespace authorization Rule within a ServiceBus Topic.
```

Comment:
would this make more sense as `Manages a ServiceBus Authorization Rule scoped at the Namespace level`?